compose/compose-v2/index.md

```diff
@@ -1,45 +1,159 @@
 ---
 description: Key features and use cases of Docker Compose
 keywords: documentation, docs, docker, compose, orchestration, containers, uses, features
-title: Compose V2 Overview
+title: Evolution of Compose
+redirect_from:
+ - /compose/cli-command-compatibility/
 ---
 
-## Compose V2 and the new `docker compose` command
+This page provides information on the history of Compose and explains the key differences between Compose V1 and Compose V2. 
 
-> Important
->
-> The new Compose V2, which supports the `compose` command as part of the Docker
-> CLI, is now available.
->
-> Compose V2 integrates compose functions into the Docker platform, continuing
-> to support most of the previous `docker-compose` features and flags. You can
-> run Compose V2 by replacing the hyphen (`-`) with a space, using `docker compose`,
-> instead of `docker-compose`.
-{: .important}
+## History
+
+The first release of Compose, written in Python, happened at the end of 2014. 
+Between 2014 and 2017 two other noticeable versions of Compose, which introduced new file format versions, were released:
 
-If you rely on using Docker Compose as `docker-compose` (with a hyphen), you can
-set up Compose V2 to act as a drop-in replacement of the previous `docker-compose`.
-Refer to the [Installing Compose](../install/index.md) section for detailed instructions.
+- [Compose 1.6.0 with file format V2](../compose-file/compose-file-v2/)
+- [Compose 1.10.0 with file format V3](../compose-file/compose-file-v3/)
 
-## Context of Docker Compose evolution
+These three key file format versions and releases prior to v1.27.0 are collectively referred to as Compose V1. 
```

Comment:
```suggestion
These three key file format versions and releases prior to v1.29.2 are collectively referred to as Compose V1. 
```
Compose V1 refer to Python version of Compose, even if versions between `v1.27.0` and `v1.29.2` could handle Compose Specification file format they are still considered as Compose V1